### PR TITLE
Chore/playwright

### LINKED ISSUE #712: Add E2E tests with Playwright

### PR DESCRIPTION
**Pull request by escapedcat**
Fixes #712

- first of all also run tests in the PRs for setup-redesign
- for some reason i18n doesn't work in unit-tests (anymore?) or just not in the app-test?
- adding playwright with a basic test and some setup and adjustments

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - name: Install backend dependencies
+      run: |
+        cd backend-mock
+        npm ci
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Run Playwright tests
+      run: npx playwright test
+    - uses: actions/upload-artifact@v4
+      if: always()
```
Comment (cstenglein) [inline .github/workflows/playwright.yml:27]: In the documentation this line is specified with `${{ !cancelled() }}`.

https://playwright.dev/docs/ci-intro#on-pushpull_request

File: tests/setup.spec.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { test } from "@playwright/test";
+
+test("happy path setup", async ({ page }) => {
```
Comment (cstenglein) [inline tests/setup.spec.ts:3]: Would rather call this something different

File: tests/setup.spec.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  await page.goto("http://localhost:3000");
+
+  // setup coice
```
Comment (cstenglein) [inline tests/setup.spec.ts:6]:
```suggestion
  // setup choice
```

File: tests/setup.spec.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  await page.getByLabel("Fresh Setup").click({ force: true }); // label is somehow covered
```
Comment (cstenglein) [inline tests/setup.spec.ts:7]: What do you mean with the comment?

File: tests/setup.spec.ts [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  await page.getByRole("button", { name: "Continue" }).click();
+
+  // delete data ingo
```
Comment (cstenglein) [inline tests/setup.spec.ts:10]: What is ingo? :)

**Comment by cstenglein** [comment]
Thank you! Just found some small things :)

File: tests/setup.spec.ts (hunk 2 above)
Comment (escapedcat) [reply tests/setup.spec.ts:3]: Open for suggestions :)

File: tests/setup.spec.ts (hunk 5 above)
Comment (escapedcat) [reply tests/setup.spec.ts:10]: Haha. no idea, copy paste I guess, will fix

File: .github/workflows/playwright.yml (hunk 1 above)
Comment (escapedcat) [reply .github/workflows/playwright.yml:27]: This was created by playwright itself during the setup... Will update

File: tests/setup.spec.ts (hunk 4 above)
Comment (escapedcat) [reply tests/setup.spec.ts:7]: For some reason the html/css setup is covering the label. But it's clearly working and works, so forcing it here seems valid 🤷

File: tests/setup.spec.ts (hunk 5 above)
Comment (escapedcat) [reply tests/setup.spec.ts]: ...remebered

**Pull request merged**